It seems that vhci_driver should use its own udev_context instance, not the one from usbip_host_common.c
--- linux-5.7/tools/usb/usbip/libsrc/vhci_driver.c.orig	2020-06-01 01:49:15.000000000 +0200
+++ linux-5.7/tools/usb/usbip/libsrc/vhci_driver.c	2020-06-21 16:02:34.325318086 +0200
@@ -15,7 +15,7 @@
 #define PROGNAME "libusbip"
 
 struct usbip_vhci_driver *vhci_driver;
-struct udev *udev_context;
+static struct udev *udev_context = NULL;
 
 static struct usbip_imported_device *
 imported_device_init(struct usbip_imported_device *idev, char *busid)
